prow/cmd/deck/main.go

```diff
@@ -1274,6 +1286,21 @@ func canTriggerJob(user string, pj prowapi.ProwJob, cfg *prowapi.RerunAuthConfig
 	}
 	if cli == nil {
 		logrus.Warning("No GitHub token was provided, so we cannot retrieve GitHub teams")
+		return false, nil
+	}
+
+	// If the job is a presubmit and has an associated PR, do the same checks as for /test
+	if pj.Spec.Type == prowapi.PresubmitJob && pj.Spec.Refs != nil && len(pj.Spec.Refs.Pulls) > 0 {
+		if pcfg == nil {
+			// If no plugins configuration is provided, skip the checks
+			logrus.Warning("No plugins configuration was provided to deck. You must provide one to reuse /test checks for rerun")
```

Comment:
nit: Do we really want to log this as a warning every time someone tries to rerun a job if the plugin config is not provided?